src/components/inputs/select-input/select-input.js

```diff
@@ -152,17 +152,18 @@ export class SelectInput extends React.Component {
      * So we need to pass null instead, so that Select clears the selected value.
      */
     return {
-      selectedOptions: props.isMulti
-        ? props.value
-            // Pass the options in the order selected by the use, so that the
-            // sorting is not lost
-            .map(value =>
-              optionsWithoutGroups.find(option => option.value === value)
-            )
-            .filter(Boolean)
-        : optionsWithoutGroups.find(
-            option => has(option, 'value') && option.value === props.value
-          ) || null,
+      selectedOptions:
+        props.isMulti && props.value
```

Comment:
needed to add && props.value because when you clear multi select now, it sends null

Reply:
going to remove this since we will wrap the breaking change